CMSIS/Core/Include/cmsis_compiler.h

```diff
@@ -53,6 +53,21 @@
  */
 #elif defined ( __ICCARM__ )
 
+  /* CMSIS compiler control architecture macros */
+  #if (__CORE__ == __ARM6M__) || (__CORE__ == __ARM6SM__)
```

Comment:
7.80.4 - ``[Fatal Error] core_cm.h@57,0: [Pe035]: #error directive: "Unknown ARM Architecture!"`` without this patch